### PR TITLE
feat: Create issues template for starter.kit

### PR DESCRIPTION
**Pull request by Zelig880**

### CODE REVIEW

File: .github/custom-issue-template/2. Add testing to the current Kit.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Background
+
+Each kit should have a provided a testing framework fully installed and cinfigured to be used
+
+# Acceptance
+
+- [ ] Add the chosen testing framework
+- [ ] Add a simple test
```
[the remark this answers is not in the document]
Comment (Zelig880) [reply .github/custom-issue-template/2. Add testing to the current Kit.md:8]: I was thinking more about the configuration and installation of it.

File: .github/custom-issue-template/3. Create a fetch component.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Background
+We want to launch each kit with an example of a component that fetches data from a data source and related tests. We have an endpoint set up as a simple hello world that accepts a greeting for example sake.
+
+# Acceptance
+- [ ] Create an API example component
+- [ ] The component should make a GET request to `https://api.starter.dev/hello?greeting=<message>` where the message is "angular-apollo-tailwind starter.dev!"
```
[the remark this answers is not in the document]
Comment (Zelig880) [reply .github/custom-issue-template/3. Create a fetch component.md:6]: yes, I am not sure if we should actually suggest GraphQl as I know that the rest API the best when trying to obtain some of the values we need. 

What is your call on this? Happy to follow your leads

File: .github/custom-issue-template/0. Generate project and establish integration point.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Background
+To make tasks parallelizable, we should create kits via integration branches to make PRs smaller and easier to read while not breaking the upstream build. Integration branches may be cleaner and easier to implement in isolation as kits are isolated to their own directory anyway.
+
+# Acceptance
+- [ ] Generate new project in the `starters/` creating a name that includes 'framework-store-style' format
+- [ ] Change `<head>` title to "framework-store-style starter kit"
```
[the remark this answers is not in the document]
Comment (Zelig880) [reply .github/custom-issue-template/0. Generate project and establish integration point.md]: Updated (sorry I copied this story from existing ones)

File: .github/custom-issue-template/3. Create a fetch component.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Zelig880) [reply .github/custom-issue-template/3. Create a fetch component.md]: I have updated the message now to include both Rest and GraphQl, but open to suggestion on the above message

File: .github/custom-issue-template/2. Add testing to the current Kit.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dustinsgoodman) [reply .github/custom-issue-template/2. Add testing to the current Kit.md:8]: Yeah - I guess it's just the "simple test" isn't needed to do the config and install. Just my 2¢. I think if you're just trying to say:

> When I run `yarn test`, I should see the tests pass on a simple `expect(true).toBe(true)` test.

File: .github/custom-issue-template/3. Create a fetch component.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dustinsgoodman) [reply .github/custom-issue-template/3. Create a fetch component.md]: I think it depends on what the team decides to use for their data fetch library. I like what you have.

File: .github/custom-issue-template/2. Add testing to the current Kit.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Zelig880) [reply .github/custom-issue-template/2. Add testing to the current Kit.md]: Sorry i missed this! :)

Changed it...

File: .github/custom-issue-template/2. Setup Storybook.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Background
+Storybook is a great way to build, validate, and test components. We want to ship a working storybook instance with each kit.
```
[the remark this answers is not in the document]
Comment (Zelig880) [reply .github/custom-issue-template/2. Setup Storybook.md:2]: I think I am going to leave the story in the Stories template and then remove it from the milestone sprint if support is not there yet. The templates are not specifically for the PODs but for any kit in general.

In the long run we may even decide to change storybook with something (i know that there are a couple of new tool on the market)

**Review by dustinsgoodman — APPROVED** [review]

**Pull request merged**